e2e/test/scenarios/embedding/embed-resource-downloads.cy.spec.ts

```diff
@@ -120,6 +127,89 @@ describeWithSnowplowEE(
           export_type: "csv",
         });
       });
+
+      describe("with dashboard parameters", () => {
+        beforeEach(() => {
+          cy.signInAsAdmin();
+
+          setTokenFeatures("all");
+
+          // Test parameter with accentuation (metabase#49118)
```

Comment:
I added a generic test, but note that with this specific filter it also test #49118. You can verify by reverting `frontend/src/metabase/redux/downloads.ts` and this test should fail (I already tested that).